packages/terra-button-group/docs/README.md

```diff
@@ -14,7 +14,18 @@
 import React from 'react';
 import ButtonGroup from 'terra-button-group';
 
-<ButtonGroup {props.......} />
+// Default button group
+  <ButtonGroup
+    buttons={[<ButtonGroup.Button text="Button 1" key="button1" />,
+      <ButtonGroup.Button text="Button 2" key="button2" />]}
+  />
```

Comment:
Can you update these to pass the ButtonGroup.Buttons as children? People will copy and paste this code and I'd rather them use children than the buttons prop.